Makefile

```diff
@@ -0,0 +1,26 @@
+IPYTHON_CONFIG := /nail/home/syedj/.ipython/profile_default
+
+.PHONY: develop-notebook
+develop-notebook: 
+	venv/bin/jupyter notebook
+
+develop:
+	venv/bin/jupyter lab --watch
+.PHONY: build
+build: venv frontend-build
+	venv/bin/pip install -I ./extension/.
+	jupyter labextension enable sparkmonitor --user           
+	jupyter serverextension enable --py --user sparkmonitor
+	ipython profile create && echo "c.InteractiveShellApp.extensions.append('sparkmonitor.kernelextension')" >>  $(IPYTHON_CONFIG)/ipython_kernel_config.py
```

Comment:
This last line is something that I'm worried about in terms of when we actually deploy to jupyterhub. I'm not entirely sure if this is going to be a problem down the line.